kernel/enter.c

```diff
@@ -663,6 +665,7 @@ int myst_enter_kernel(myst_kernel_args_t* args)
     myst_process_t* process = NULL;
     myst_fstype_t fstype;
     int tmp_ret;
+    int create_appenv_ret;
```

Comment:
I think you can reuse `tmp_ret` instead of creating a new variable.

Reply:
I think it might be better to use a separate variable in case the value of tmp_ret is altered by some other part of the code